src/helpers/utils.ts

```diff
@@ -132,3 +137,14 @@ export function getIpfsUrl(url) {
     import.meta.env.VITE_IPFS_GATEWAY || 'cloudflare-ipfs.com';
   return getUrl(url, gateway);
 }
+
+export function relativePeriod(state: any, start: any, end: any): any {
```

Comment:
I think this shouldn't be here in utils. Composables can use utils but not the other way around. Utils should be stateless. I'd probably put this in the `useIntl` composable.

Reply:
Yes that makes more sense, thanks 🙂